ddl/foreign_key_test.go

```diff
@@ -155,6 +155,8 @@ func (s *testForeighKeySuite) TestForeignKey(c *C) {
 		checkOK = true
 	}
 	d.SetHook(tc)
+	originalHook := d.GetHook()
```

Comment:
This should put before line#157.

Reply:
Sorry for that.
At begiing , I would have thought I should make tc to restore original.
And IMO,the original is before `tc`.